frontend/public/components/utils/dropdown.jsx

```diff
@@ -285,7 +285,18 @@ class Dropdown_ extends DropdownMixin {
   applyTextFilter_(autocompleteText, items) {
     const { autocompleteFilter } = this.props;
     if (autocompleteFilter && !_.isEmpty(autocompleteText)) {
-      items = _.pickBy(items, (item, key) => autocompleteFilter(autocompleteText, item, key));
+      const newItems = [];
+      let totalItems = _.map(items, (item) => item.props);
+      totalItems = autocompleteFilter(autocompleteText, totalItems);
+      _.map(totalItems, (filterItem) => {
```

Comment:
Why call `_.map` and not use the result? Should this just be `forEach`?

Reply:
Done